File: Leetcode/2348 - Number of Zero-Filled Subarrays/solution.py
def zeroFilledSubarray(self, nums: List[int]) -> int:
    totCount = 0
    count = 0

    for num in nums:
        if num == 0:
            count += 1
        else:
            totCount += (count * (count + 1)) // 2
            count = 0

    totCount += (count * (count + 1)) // 2
    return totCount

File: Leetcode/2348 - Number of Zero-Filled Subarrays/solution.cpp
long long zeroFilledSubarray(vector<int>& nums) {
    long long totCount = 0, count = 0;

    for (int i = 0; i < nums.size(); i++) {
        if (nums[i] == 0) {
            count++;
        } else {
            totCount += ((count * (count + 1)) / 2);
            count = 0;
        }
    }
    totCount += ((count * (count + 1)) / 2);
    
    return totCount;
}